### DISCUSSION TITLE
Is there some way to reposition [8/11] style search indicator?

### DISCUSSION THREAD

**Discussion by shmerl** [neovim/neovim#29662] [Q&A] [answered]
When doing search in neovim, it shows "8 out of 11" type of indicator as: [8/11] on the right side of the command line. Is there some way to reposition it more to the left, closer to the actual search pattern? It's annoying to look so far to the right to track it.

Can it be done with some configuration, and if not, are there some options to programmatically do it?

**Comment by zeertzjq** [comment, marked as answer]
See `:h searchcount()` for how to programmatically do it.

**Reply by shmerl** [reply]
Ah, thanks for the pointer! So if I understand correctly, I need to first disable the default by adding `S` to `shortmess` setting string and then figure out how to display the count somewhere using results from `searchcount`.

**Comment by Shougo** [comment]
It is shortmess feature.

Please see `:help shm-S`.

**Reply by Shougo** [reply]
It is builtin feature and you cannot change the position.
But you can implement the feature by script.
